components/auth/Join.tsx

```diff
@@ -104,7 +104,7 @@ const Join = () => {
             <span className="checkbox-toggle"></span>
           </div>
           <label className="label">
-            <span className="label-text">Agree to <Link href='/terms-condition' className='text-primary' target="_blank">Terms and conditions</Link></span>
+            <span className="label-text">Agree to <Link href={`${process.env.NEXT_PUBLIC_TERMS_AND_CONDITIONS_URL}`} className='text-primary' target="_blank">Terms and conditions</Link></span>
```

Comment:
Can we hide the terms and condition checkbox if `NEXT_PUBLIC_TERMS_AND_CONDITIONS_URL` is not set? This way it won't break for existing users of the starter kit.